bundles/cxx_remote_services/CMakeLists.txt

```diff
@@ -15,21 +15,14 @@
 # specific language governing permissions and limitations
 # under the License.
 
-celix_subproject(REMOTE_SERVICE_ADMIN "Option to enable building the C++17 Remote Service Admin Service bundles" OFF)
-if (REMOTE_SERVICE_ADMIN)
+celix_subproject(CXX_REMOTE_SERVICE_ADMIN "Option to enable building the C++17 Remote Service Admin Service bundles" ON)
```

Comment:
For experimental features, it really should default to OFF. I don' want to change CI behavior.

Reply:
Fixed